streamparse/cmdln.py

```diff
@@ -79,8 +83,11 @@ def main():
     if args["run"]:
         time = int(args["--time"])
         par = int(args["--par"])
+        ackers = int(args["--ackers"]) if args.get("--ackers") else par
```

Comment:
Good catch, had `int(args["--ackers"]) if args["--ackers"] else par` initially but got a key error when not specified, so switched it to `.get()` and didn't realize that it became redundant. Fixed in 0b57485